attendance/API_HR/c-hr.php

```diff
@@ -2434,6 +2432,42 @@ public function sendWelcomeMail($userID) {
 
     //end New Employee Welcome Email
 
+    //Leave of New Employee
+
+    public static function leaveOfNewEmployee($userID) {
+        $r_message3 = "";
+        $userInfo = self::getUserInfo($userID);
+        $date_of_joining = $userInfo['dateofjoining'];
+        $d = new DateTime($date_of_joining);
+        $d->modify('first day of this month');
+        $firstDay = $d->format('Y-m-d');
```

Comment:
this is not the way to get first day of the month, there a generic function which gives us generic days summary of office.
i explained you in the morning

Reply:
function name is : getGenericMonthSummary